recipes/embree/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+
+import os
+from conans import CMake, tools
+from conans.model.conan_file import ConanFile
+
+
+class Embree(ConanFile):
+    name = "embree"
+    version = "3.6.1"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "embree", "raytracing", "rendering")
+    description = "Intel's collection of high-performance ray tracing kernels."
+    generators = "cmake"
+    homepage = "https://embree.github.io/"
+    _source_folder = "source_subfolder"
+    _build_folder = "build_subfolder"
+    exports_sources = "CMakeLists.txt"
+    settings = "os", "compiler", "build_type", "arch"
+    requires = "tbb/2019_u9"
+
+    options = {
+        "fPIC": [
+            True,
+            False
+        ],
+        "backface_culling": [
+            True,
+            False
+        ],
+        "ignore_invalid_rays": [
+            True,
+            False
+        ],
+        "ray_masking": [
+            True,
+            False
+        ]
+    }
+
+    default_options = {"fPIC": True,
+                       "backface_culling": False,
+                       "ignore_invalid_rays": False,
+                       "ray_masking": False,
+                       "tbb:tbbmalloc": True,
+                       }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+
+        tools.get(**self.conan_data["sources"][self.version])
+        target_name = "{0}-{1}".format(self.name, self.version)
+        os.rename(target_name, self._source_folder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+
+        # Configure CMake library build:
+        cmake.definitions["CMAKE_INSTALL_PREFIX"] = self.package_folder
+        cmake.definitions["EMBREE_STATIC_LIB"] = False
+        cmake.definitions["BUILD_TESTING"] = False
+        cmake.definitions["EMBREE_TUTORIALS"] = False
+        cmake.definitions["EMBREE_GEOMETRY_CURVE"] = True
+        cmake.definitions["EMBREE_GEOMETRY_GRID"] = True
+        cmake.definitions["EMBREE_GEOMETRY_INSTANCE"] = True
+        cmake.definitions["EMBREE_GEOMETRY_QUAD"] = True
+        cmake.definitions["EMBREE_GEOMETRY_SUBDIVISION"] = True
+        cmake.definitions["EMBREE_GEOMETRY_TRIANGLE"] = True
+        cmake.definitions["EMBREE_GEOMETRY_USER"] = True
+        cmake.definitions["EMBREE_RAY_PACKETS"] = True
+        cmake.definitions["EMBREE_RAY_MASK"] = self.options.ray_masking
+        cmake.definitions["EMBREE_BACKFACE_CULLING"] = self.options.backface_culling
+        cmake.definitions["EMBREE_IGNORE_INVALID_RAYS"] = self.options.ignore_invalid_rays
+        cmake.definitions["EMBREE_ISPC_SUPPORT"] = False
+
+        if self.settings.os != "Windows":
+            cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.fPIC
```

Comment:
Conan automatically sets this once through the `CMake` build helper when your recipe contains the `fPIC` option, you don't need to pass it manually.

Reply:
line removed.